test/topology/test_cluster_features.py

```diff
@@ -192,6 +192,7 @@ async def test_partial_upgrade_can_be_finished_with_removenode(manager: ManagerC
         await manager.server_sees_others(srv.server_id, len(servers) - 1)
 
     # The feature should not be enabled yet on any node
+    cql = manager.cql
```

Comment:
You can use `get_cql()` so mypy doesn't complain (because `manager.cql` can be `None`)